server/migrations/20170303151657-create-reminders.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+module.exports = {
+  up: function(queryInterface, Sequelize) {
+    return queryInterface.createTable('Reminders', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      status: {
+        type: Sequelize.INTEGER
+      },
+      timeZone: {
+        allowNull: false,
+        type: Sequelize.STRING
+      },
+      Repeat: {
+        allowNull: false,
+        type: Sequelize.STRING
+      },
+      Expired: {
+        allowNull: false,
+        type: Sequelize.STRING
+      },
+      deliverAt: {
+        allowNull: false,
+        type: Sequelize.STRING
+      },
+      lastDeliveredAt: {
+        allowNull: false,
```

Comment:
I would think that this field would be initially null, to indicate that the notification has never been delivered!